### DISCUSSION TITLE
Which tests are used for grading?

### DISCUSSION THREAD

**Discussion by tdiyar** [ANLAB-KAIST/KENSv3#121] [Project Q&A 2. Transport Layer (TCP)]
Pdf file of Programming Assignment 2 description says that only the following test files will be used for grading:
- ./app/kens/testopen.cpp (test-kens-open)
- ./app/kens/testbind.cpp (test-kens-bind)
- ./app/kens/testhandshake.cpp (test-kens-handshake)
- ./app/kens/testclose.cpp (test-kens-close)

Does it mean that mean that we don't need to test transfer of data (test-kens-transfer)? So we don't need to implement read and write syscalls at all?

**Comment by rubidam** [comment]
yes, I think that is PA3 part.
